File: Ej 14 (Factorial)/Python/FactRec.py
## DeathbatO

def fact_rec(num):
    if num == 0:
        return 1
    else:
        return num * fact_rec(num-1)

for i in range(5):
    num = int(input("Ingrese un numero para calcular su factorial: "))
    if num > 0:
        print(f"El facorial de {num} es {fact_rec(num)}")
    elif num == 0:
        print("El factorial de 0 es 1")
    else:
        print("ERROR: El numero es negativo")

File: Ej 14 (Factorial)/Java/FactRec.java
// DeathbatO

import java.util.Scanner;

public class FactRec {
	
	public static int Factorial(int n){
		if(n == 0){
            return 1;
        }else{
            return n * Factorial(n-1);
        }
	}
	
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int num;
		for(int i = 0; i < 5; i++){
			System.out.print("Ingrese un numero para calcular su factorial: ");
			num = sc.nextInt(); sc.nextLine();
			if(num > 0){
				System.out.println("El factorial de "+num+" es: "+Factorial(num));
			}else if(num == 0){
				System.out.println("El factorial de 0 es: 1");
			}else{
				System.out.println("ERROR: Numero negativo");
			}
		}
        sc.close();
	}
}
